chapter2/tests/t19/terraform/_modules/container/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "cont" {
```

Comment:
invalid non semantic name for the variable.
it should be prefixed with `container_` prefix

Reply:
> invalid non semantic name for the variable. it should be prefixed with `container_` prefix

@dmitry-mightydevops
Done